cranelift/isle/docs/language-reference.md

```diff
@@ -1442,12 +1442,14 @@ The grammar accepted by the parser is as follows:
                | "\t"
                | "\n"
                | "\r"
+               | "\b"
+               | "\f"
```

Comment:
for sake of completeness. I suppose they're also ignored by the parser?

Reply:
Actually no -- the full list of whitespace characters is [here](https://github.com/bytecodealliance/wasmtime/blob/41ff8e7b6b47ffca748db6fc5ec433d6c69fe486/cranelift/isle/isle/src/lexer.rs#L114). The original list was correct.

Reply:
Should have double checked first, sorry about that.